amundsen_application/static/js/ducks/notification/reducer.ts

```diff
@@ -34,22 +34,31 @@ export function submitNotificationSuccess(): SubmitNotificationResponse {
 export function closeRequestDescriptionDialog(): ToggleRequestAction {
   return {
     type: ToggleRequest.CLOSE,
+    payload: {
+      checkedInputs: [],
+    }
   };
 };
 
-export function openRequestDescriptionDialog(): ToggleRequestAction {
+export function openRequestDescriptionDialog(checkedInputs?: string[]): ToggleRequestAction {
+  checkedInputs = checkedInputs || [];
```

Comment:
Rather than an array called `checkedInputs`, I would suggest using a string called `requestType`. There isn't a current use case for passing multiple values here, so we don't need to make this an array. The naming also seems to imply the mapping from request type to checked inputs is the responsibility of the caller. I think it should be the component that converts `requestType` and maps it to specific checkboxes. 

If we did have a use for checking both boxes, we could create a new `requestType` that maps to both inputs, rather than having the caller send `['column-description', 'table-description']`.

-------
We could consider adding another parameter to specify the exact column or table that was requested. Currently we get the table from the global state but we don't know which column.

```
openRequestDescriptionDialog(requestType: string, table: TableMetadata, column?: ColumnMetadata)
```

Reply:
1. Up for changing `checkedInputs` to some enum string.
2. I'll pas the column index since the component already has access to the whole `TableMetadata`